### DISCUSSION TITLE
How can I make the menu bar show again?

### DISCUSSION THREAD

**Discussion by yhwwwwww** [kingToolbox/WindTerm#400] [Q&A] [answered]
I hided the menu bar, but now I want to see it again. But how can I make it show?

**Comment by uuuhuuu** [comment]
>>    alt + M

**Comment by kingToolbox** [comment, marked as answer]
Sorry for the late reply, I don't know why I missed this question at the time.

You can use `Alt+W, Alt+M` to show the menubar again, `W` means `Windows` or `Widget` and `M` means `MenuBar`. I hope there will be a better solution in the future, but I haven't found a good design yet.

**Reply by izhx** [reply]
😀 Hi, on macOS, could you put the menu in the system menu bar, just like other macOS applications? Just a suggestion.

**Comment by fanybook** [comment]
@kingToolbox on macOS, can't `Alt+W, Alt+M`


https://github.com/kingToolbox/WindTerm/issues/1821
https://github.com/kingToolbox/WindTerm/issues/2698
